pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -274,3 +277,18 @@ func validateTaskTemplate(task v1alpha1.TaskSpec, job *v1alpha1.Job, index int)
 
 	return ""
 }
+
+func validatePodNameLength(template v1.PodTemplateSpec, job *v1alpha1.Job, index int) string {
```

Comment:
I'm ok about the validation. Perhaps strict validation just as k8s resource rule is better.

Reply:
I have rename method to `validateK8sPodNameLength` and `validateK8sSvcNameLength`.What do you think?

Reply:
> I have rename method to `validateK8sPodNameLength` and `validateK8sSvcNameLength`.What do you think?

OK. Can you give a stricter rule validation? For example, the least length/start character of name/character category of name.

Reply:
I have optimize code.